pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -358,6 +353,11 @@ func generateConfig(k8s config.KubernetesConfig) (string, error) {
 		EtcdDataDir:       "/data/minikube", //TODO(r2d4): change to something else persisted
 		NodeName:          k8s.NodeName,
 		ExtraArgs:         extraComponentConfig,
+		NoTaintMaster:     false,
+	}
+
+	if version.GTE(semver.MustParse("1.10.0-alpha.0")) {
```

Comment:
nit: maybe add a comment here explaining why NoTaintMaster is required for >=1.10.0-alpha.0